src/test/java/com/saebyeok/saebyeok/service/ReportServiceTest.java

```diff
@@ -50,4 +73,27 @@ void getReportCategories() {
         assertThat(reportCategoryResponses.get(0).getName()).isEqualTo(CATEGORY_NAME);
         assertThat(reportCategoryResponses.get(0).getContent()).isEqualTo(CATEGORY_CONTENT);
     }
+
+    @DisplayName("게시물 신고 생성을 요청하면 게시물 신고가 생성된다")
+    @Test
+    void createReportTest() {
+        ReportCreateRequest createRequest = new ReportCreateRequest(REPORT_CONTENT, REPORTED_ID, CATEGORY_ID, ReportType.ARTICLE.getName());
+        when(reportRepository.save(any(Report.class))).thenReturn(report);
+        when(reportCategoryRepository.findById(1L)).thenReturn(Optional.of(reportCategory));
+
+        Report createdReport = reportService.createReport(member, createRequest);
+
+        assertThat(createdReport).isNotNull();
```

Comment:
```suggestion
        assertThat(createdReport).isNotNull()
                                 .isInstanceOf(Report.class);
```
요렇게 Report class의 인스턴스가 맞는지 검증이 추가되면 쬐꿈이나마 더 튼튼해보이지 않을까 해서 남겨보아용 ㅎㅎ

Reply:
감사합니다!! 바로 적용할게요 크...